boilerplate/templates/index.html

```diff
@@ -272,23 +272,19 @@ <h3>Bookend Sample</h3>
          role="button"
          on="tap:boilerplate.scrollTo(duration=200)"
          class="fab">View Code</div>
-    <amp-analytics type="googleanalytics" id="analytics">
-      <script type="application/json">
-        {
-          "vars": {
-            "account": "UA-73836974-1"
-          },
-          "triggers": {
```

Comment:
Did we miss porting over `triggers` over to the new setup/

Reply:
I thought that this is not really needed as page views are tracked by default. Or am I wrong about this?

Reply:
We discussed this offline.